docs/how_to_knit/basics.md

```diff
@@ -1,22 +1,35 @@
 
 ## Single Bed Fair Isle
-For your first practice piece, load a test pattern such as:
+
+To create your own patterns, refer to the AYAB manual, section "pattern_image_creation". Make sure the 
```

Comment:
I propose to insert a link rather than describing where to go. For information about inserting links, see https://www.mkdocs.org/user-guide/writing-your-docs/#internal-links